service/src/test/java/com/fitgoal/service/LoginServiceImplTest.java

```diff
@@ -0,0 +1,76 @@
+package com.fitgoal.service;
+
+import com.fitgoal.api.LoginService;
+import com.fitgoal.api.domain.User;
+import com.fitgoal.api.domain.UserLoginData;
+import com.fitgoal.api.exceptions.IncorrectEmailOrPasswordException;
+import com.fitgoal.dao.UserDao;
+import com.fitgoal.dao.domain.UserDto;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.Optional;
+
+import static com.fitgoal.service.util.TestHelper.createUserDto;
+import static com.fitgoal.service.util.TestHelper.createUserLoginData;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@RunWith(MockitoJUnitRunner.class)
+public class LoginServiceImplTest {
+
+    @Mock
+    private UserDao userDao;
+
+    private LoginService loginService;
```

Comment:
you already using the MockitoJUnitRunner, it means you can inject the mocked dependencies using @InjectMocks. And no need to have a setUp method

Reply:
Done